lighthouse-core/audits/is-on-https.js

```diff
@@ -72,6 +72,11 @@ class HTTPS extends Audit {
       extendedInfo: {
         formatter: Formatter.SUPPORTED_FORMATS.URL_LIST,
         value: insecureRecords
+      },
+      details: {
```

Comment:
On interesting, granularity is good. I was thinking devtools would replace entire methods (`_renderTable`) with its own implementation or call us and annotate after the fact. Still waiting on answers to how the latter works :)